activities/beginner/using-fetch/src/example/script.js

```diff
@@ -0,0 +1,11 @@
+fetch('https://jsonplaceholder.typicode.com/todos/1')
```

Comment:
Do you think we should explicitly describe that the `fetch` function takes in a URL?